app/tests/spec/views/mixins/password-mixin.js

```diff
@@ -120,18 +120,24 @@ define(function (require, exports, module) {
     });
 
     describe('show passwordHelper', function () {
-      it('set warning opacity to 1 if password length is less than 8', function () {
-        view.$('.password').val('1234');
-        view.showPasswordHelper();
+      it('set warning opacity to 1 if any password length is less than 8', function () {
+        view.$('#password').val('1234');
+        view.$('#old_password').val('123456789');
+        view.$('#new_password').val('123456789');
```

Comment:
I don't believe we need those extra tests after adding @shane-tomlinson recommendation for checking all password classes.